### PR TITLE
Feature/datahub 10 17 release

### PR DESCRIPTION
**Pull request by cheinz-sag**
Hi,

can you pls review? The stuff is to be merged to develop and 10.17. I'd propose to wait until Tim has completed his review as he'll check also contents.

Thanks
Christoph

### CODE REVIEW

File: content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-weight: 50
+weight: 60
 title: Monitoring offloading jobs
 layout: redirect
 ---
 
-Once you have configured and started your offloading pipelines, they regularly offload data to the data lake. {{< product-c8y-iot >}} DataHub provides insights into the execution status of the different pipelines so that you can investigate whether everything is running as expected.
+Once you have configured and started your offloading pipelines, they regularly offload data to the data lake. {{< product-c8y-iot >}} DataHub UI provides insights into the execution status of the different pipelines so that you can investigate whether everything is running as expected. For the case of offloading failures, you can also configure the offloading pipeline to raise an alarm as described in [Raising alarms](/datahub/configuring-offloading-jobs/#raising-alarms).
```
[the remark this answers is not in the document]
Comment (cheinz-sag) [reply content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md:7]: Done

File: content/datahub/working-with-datahub-bundle/aligning-data-modeling-and-offloading.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+title: Aligning data modeling and offloading
+layout: redirect
+---
+
+#### Mapping document data to relational data
+
+{{< product-c8y-iot >}} DataHub allows to offload data from the {{< product-c8y-iot >}} platform into a data lake and the subsequent analysis of the offloaded data using SQL. For that purpose, {{< product-c8y-iot >}} DataHub must transform the data from the document-based format into a relational format, which is then persisted as Parquet files in the data lake.
+
+When the offloading is configured, data from the documents in the operational database of the {{< product-c8y-iot >}} platform is transformed and stored in columns of the target table in the data lake. The system either automatically generates those transformations or proposes them to the user with the option to modify them. The user can also configure additional transformations. A configuration defining how a data field is transformed into a column comprises:
+- The expression how to retrieve the data from the source document
```
[the remark this answers is not in the document]
Comment (cheinz-sag) [reply content/datahub/working-with-datahub-bundle/aligning-data-modeling-and-offloading.md:12]: Done

File: content/datahub/working-with-datahub-bundle/aligning-data-modeling-and-offloading.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- The name of the target column
+- The type of the target column
```
[the remark this answers is not in the document]
Comment (cheinz-sag) [reply content/datahub/working-with-datahub-bundle/aligning-data-modeling-and-offloading.md:14]: Done

File: content/datahub/working-with-datahub-bundle/aligning-data-modeling-and-offloading.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+layout: redirect
+---
+
+#### Mapping document data to relational data
+
+{{< product-c8y-iot >}} DataHub allows to offload data from the {{< product-c8y-iot >}} platform into a data lake and the subsequent analysis of the offloaded data using SQL. For that purpose, {{< product-c8y-iot >}} DataHub must transform the data from the document-based format into a relational format, which is then persisted as Parquet files in the data lake.
+
+When the offloading is configured, data from the documents in the operational database of the {{< product-c8y-iot >}} platform is transformed and stored in columns of the target table in the data lake. The system either automatically generates those transformations or proposes them to the user with the option to modify them. The user can also configure additional transformations. A configuration defining how a data field is transformed into a column comprises:
+- The expression how to retrieve the data from the source document
+- The name of the target column
```
[the remark this answers is not in the document]
Comment (cheinz-sag) [reply content/datahub/working-with-datahub-bundle/aligning-data-modeling-and-offloading.md:13]: Done

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md [hunk 5]
```diff
@@ -32,7 +32,11 @@ During the initial configuration of {{< product-c8y-iot >}} DataHub, a so-called
...
 
-Having the corresponding permission assigned, the user can grant other Dremio users, which do not necessarily relate to {{< product-c8y-iot >}} DataHub, different permissions on the data lake source or the space, for example, for reading data from the data lake or creating a table in the data lake. Granting permissions to other users should be done carefully in order to avoid that sensitive information is exposed to the wrong users. 
+Having the corresponding permission assigned, the user can grant other Dremio users, which do not necessarily relate to {{< product-c8y-iot >}} DataHub, different permissions on the data lake source or the space, for example, for reading data from the data lake or creating a table in the data lake. 
+
+{{< c8y-admon-caution >}}
+Granting permissions to other users should be done very carefully in order to avoid that sensitive information is exposed to the wrong users. In particular, permissions should never be granted to all users as in that case each Dremio user can access the data lake source or space respectively.
```
Comment (doernemt) [inline content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:38]: ... ALL Dremio userS of/in the Cumulocity instance can access ...

File: content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md [hunk 6]
```diff
@@ -33,7 +33,9 @@ The system is configured to keep a limited history of the last job executions.
 
 Click **Reload** to refresh the list.
 
-You can filter the entries by their status or timestamp by using the filter controls at the top. The pagination buttons at the bottom can be used to traverse the history entries.
+You can filter the entries by their status or timestamp by using the filter controls at the top. Click **Apply** to filter entries with the current filter settings. Click **Reset filter** to reset the current filter settings.
+
+The pagination buttons at the bottom can be used to traverse the history entries.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md:38]:
```suggestion
The page navigation buttons at the bottom can be used to traverse the history entries.
```

[the remark this answers is not in the document]
Comment (cheinz-sag) [reply content/datahub/working-with-datahub-bundle/monitoring-offloading-jobs.md:38]: Done

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md (hunk 5 above)
Comment (cheinz-sag) [reply content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:38]: Done

File: content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md [hunk 7]
```diff
@@ -52,7 +52,7 @@ The following types of data lakes are currently supported:
 While the other settings are fixed once the initial configuration was saved, the authentication type as well as the values of the selected authentication type can be changed afterwards. Click **Edit**, set new values, and either click **Save credentials** to save the update or **Cancel** to keep the old value.
 
 {{< c8y-admon-info >}}
-Note that the account type must be **StorageV2**, and the **Hierarchical namespace** feature must be activated for the corresponding Azure Storage account. It is for performance reasons recommended to set the **Blob access tier** to **Hot**.
+Note that the account type must be **StorageV2**, and the **Hierarchical namespace** feature must be activated for the corresponding Azure Storage account. It is for performance reasons recommended to set the **Blob access tier** to **Hot**. Also note that in case IP white-listing is activated, Cumulocity IoT DataHub cannot access the data lake.
```
[the remark this answers is not in the document]
Comment (cheinz-sag) [reply content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md:55]: Done

**Review by MWindel — APPROVED** [review]

**Comment by MWindel** [comment]
Merged to develop, and cherry-picked to release/r10.17.0

**Pull request merged**